packages/backend/src/routers/organizationRouter.ts

```diff
@@ -65,6 +65,21 @@ organizationRouter.post('/projects', isAuthenticated, async (req, res, next) =>
         .catch(next),
 );
 
+organizationRouter.delete(
+    '/projects/:projectUuid',
+    isAuthenticated,
```

Comment:
We should also add `unauthorisedInDemo,` to block this route for demo deployments.